content/blog/2023-07-07-newsletter.md

```diff
@@ -0,0 +1,181 @@
+
+# Videos
+
+- [Libervia: the Universal Communication Ecosystem](https://www.yewtu.be/watch?v=EkkiKn0TONs)
+
+## Articles
+
+- [A new place for poezio and slixmpp](https://blog.mathieui.net/en/new-home-for-poezio-slixmpp.html).
+- [Elbe-Sprint Hamburg 2023: Post-Sprint Summary](https://xmpp.org/2023/08/elbe-sprint-hamburg-2023-post-sprint-summary/)
+- OpenPGP & PGPainless
+  - [Creating an OpenPGP Web-of-Trust Implementation – Knitting a Net](
+  - [PGPainless meets the Web-of-Trust](https://blog.jabberhead.tk/2023/07/25/pgpainless-meets-the-web-of-trust/)
+- Two thoughts on killing decentralized networks:
+  - [How to Kill a Decentralised Network (such as the Fediverse)](https://ploum.net/2023-06-23-how-to-kill-decentralised-networks.html)
+  - [A thread on this article by MattJ](https://floss.social/@mattj/110595483224579575)
+- [XMPP - Teil 2 - XMPP etwas genauer](https://anoxinon.media/blog/xmpp2/) [DE]
+
+## Software news
+### Clients and applications
+
+- [JMP is launched and out of Beta](https://blog.jmp.chat/b/launch-2023)
+- [Cheogram Android version `2.12.1-6`](https://git.singpolyma.net/cheogram-android/refs/2.12.1-6) adds per account colours, per account quiet hours, thumbhash previews of un-downloaded files, and more.
+- [Libervia progress note 2023-W22](https://www.goffi.org/b/libervia-progress-note-2023-w22-x3Wa) on A/V Implementation, ActivityPub Gateway and more.
+- [Dino 0.4.3](https://github.com/dino/dino/releases/tag/v0.4.3).
+- [Movim 0.22.2 is out](https://mov.im/node/pubsub.movim.eu/Movim/1d407d21-989d-404b-9a39-aa9b328882d1).
+
+### Servers
+
+- The ejabberd [mod_s3_upload](https://github.com/processone/ejabberd-contrib/tree/master/mod_s3_upload) module gained [support for the use of a separate download host](https://github.com/processone/ejabberd-contrib/pull/320). This allows clients to download media content from a statically hosted S3 bucket. This was initially proposed to allow ejabberd to integrate 
+with [Garage](https://garagehq.deuxfleurs.fr/), an open-source distributed object storage service tailored for self-hosting.
+
+### Libraries & Tools
+
+- New website and [xmpp.rs version 0.4 release](https://xmpp.rs/blog/release-xmpp-0-4/): After more than 3 years and half since the last release (version 0.3 in septembre 2019), xmpp-rs developers are happy to announce xmpp-rs version 0.4.0. If you don't know about xmpp-rs, those are Rust crates for interoperable and federated instant messaging and other social networking applications.
```

Comment:
```suggestion
- New website and [xmpp.rs version 0.4 release](https://xmpp.rs/blog/release-xmpp-0-4/): After more than 3 and a half years since the last release (version 0.3 in septembre 2019), xmpp-rs developers are happy to announce xmpp-rs version 0.4.0. If you don't know about xmpp-rs, those are Rust crates for interoperable and federated instant messaging and other social networking applications.
```